aws/sqs/sqs_integration_test.go

```diff
@@ -256,16 +256,17 @@ func TestSQSSendWithDelay(t *testing.T) {
 	require.Nil(t, err, fmt.Sprintf("Error creating sqs client: %v", err))
 
 	// ACTION
-	err = client.SendWithDelay(awsCmdQueueURL(), testMessage, 4) // delay seconds
-	time.Sleep(2 * time.Second)
-	messageCountBeforeDelay := awsCmdQueueCount()
-	time.Sleep(3 * time.Second)
-	messageCountAfterDelay := awsCmdQueueCount()
+	err = client.SendWithDelay(awsCmdQueueURL(), testMessage, 5) // delay seconds
+
+	start := time.Now()
+	for awsCmdQueueCount() < 1 {
```

Comment:
If the "send" part of `SendWithDelay` fails, the test gets stuck here forever.

Reply:
Good point, will put in a check